xcp_d/utils/restingstate.py

```diff
@@ -178,10 +197,22 @@ def compute_alff(data_matrix, low_pass, high_pass, TR, sample_mask=None):
         # from the value closest to the low pass cutoff, to the value closest
         # to the high pass pass cutoff
         alff[i_voxel] = len(ff_alff) * np.mean(power_spectrum_sqrt[ff_alff[0] : ff_alff[1]])
+        falff[i_voxel] = np.sum(power_spectrum_sqrt[ff_alff[0] : ff_alff[1]]) / np.sum(
+            power_spectrum_sqrt
+        )
+        peraf[i_voxel] = (
+            np.mean(np.abs((voxel_data_for_peraf - mean_matrix[i_voxel]) / mean_matrix[i_voxel]))
+            * 100
+        )
+
+        tsnr[i_voxel] = mean_matrix[i_voxel] / np.std(voxel_data_for_peraf)
 
     assert alff.size == n_voxels, f"{alff.shape} != {n_voxels}"
 
-    # Add second dimension to array
+    # Add second dimension to arrays
```

Comment:
I like to use `.reshape(-1, 1)` for padding, but I don't think there's an official best way to do it

Reply:
I actually think the Numpy-preferred method might be [`numpy.expand_dims`](https://numpy.org/doc/stable/reference/generated/numpy.expand_dims.html), but I don't feel strongly about it.